src/runtime/services.ts

```diff
@@ -27,7 +47,7 @@ export class Services {
   }
 }
 
-Services.registry = {};
+Object.freeze(Services);
```

Comment:
this is going to make testing hard, isn't it?

Reply:
hmm... likely. 
I'll make another PR in a bit with tests (since @sjmiles by now has a few example use cases of this services functionality, if I understand everything correctly). 

Should this get in the way, I'll warm the object back up.